site-src/index.md

```diff
@@ -94,8 +94,32 @@ This flexibility allows the API to adapt to vastly different
 organizational models and implementations while remaining a portable and
 standard API.
 
+## What's the difference between Gateway API and an API Gateway?
 
-## Who is working on Gateway?
+The concept of a Gateway is a networking component that allows data to transfer from one
+network to another network.
+
```

Comment:
I'd leave this section out because at least in Gateway API, Gateways don't actually need to cross networks. I think the next sentence works well as an intro to this section.